### DISCUSSION TITLE
is it possible to access page "metadata" SEO in the template?

### DISCUSSION THREAD

**Discussion by lior-amsalem** [vercel/next.js#47824] [Help]
### Summary

in the top level ("app" approach) i store `<h1>` with title of the page and want to access "metadata" provided by a lower level page.tsx file.

is it possible? i tried via props but couldn't find anything

**Comment by rdyar** [comment]
I too would like to be able to access the page metadata - especially from within a layout file.

I'd like to be able to show a small div when in development that shows the title and the description along with the character count for both - on each page.

 I can do this on each page, but on top of being a lot of duplication it won't show the correct info if there is a template(?). So I would like to have a small component in the layout file that has access to the page metadata and then it could show it to me on each page.

If I add this to the layout file as it is all I get is the metadata that is default set on the layout itself - I want to be able to see what is set on each page.

I tried looking on the childrens props but don't see anything about the metadata.
